sql/dql/basic-queries/select.md

```diff
@@ -31,23 +31,45 @@ aspects:
 ---
 ## Content
 
-To get all of the data from a table, you can use a `SELECT` statement.
+Consider the following table named `pokemon`:
+
+| id | name       | total | hp | attack | defense |
+|----|------------|-------|----|--------|---------|
+| 10 | Caterpie   | 195   | 45 | 30     | 35      |
+| 11 | Metapod    | 205   | 50 | 20     | 55      |
+| 12 | Butterfree | 395   | 60 | 45     | 50      |
+| 13 | Weedle     | 195   | 40 | 35     | 30      |
+| 14 | Kakuna     | 205   | 45 | 25     | 50      |
+| 15 | Beedrill   | 395   | 65 | 90     | 40      |
+
+To get all of the data from a table, you can use a `SELECT *` statement.
 
 ```sql
 SELECT *
 FROM pokemon;
 ```
 
-The above command gets all of the *columns* and all of the *rows* from the `pokemon` table.
+The above command gets all of the *columns* and all of the *rows* from the `pokemon` table[1].
 
-We can also use the `SELECT` statement to specify the exact *columns* we want to keep from the table, and to discard the others.
+We can also use the `SELECT` statement to specify the exact *columns* we want to display from the table. Depending on how the database was created, you might have to deal with complicated column names. In order to increase readability, or to avoid repeatedly writing lengthy names, the columns can be renamed using *aliases* represented by the `AS` keyword.
```

Comment:
after "We can also use the `SELECT` statement to specify the exact *columns* we want to display from the table."

there should be a quick example (no need to show output)

then start the next paragraph "Depending on how the database was created, you might have to deal with complicated column na...."

Reply:
Added quick example.